teams/ruby_services.md

```diff
@@ -129,16 +129,14 @@ test_story = build(:story, :published, in_print: false)
 
 The issue we have found is that as time goes on, the required conditions of the object may change, and then there is contention as to whether the existing trait gets updated or the test code.
 
-To avoid this our agreed pattern is that *all object setup should be in a trait*.
+We have then decided that every factory will have a base status of data. Any test that requires specific data to be there which are different from the `default` status of the object will then have to define those in a Trait.
```

Comment:
```suggestion
Every factory should include the essential data by default. If a test needs specific data that differs from the `default` status of the object, it should use a trait to define the data.
```

I thought "a base status of data" could be a bit clearer, so tried to reword it and ended up with this – what do you think?

Reply:
Sounds much better!